### DISCUSSION TITLE
Device : Oneplus 7t

### DISCUSSION THREAD

**Discussion by ghost** [Divested-Mobile/DivestOS-Build#24] [General] [answered]
Hi there , I tried the build for oneplus 7t and I found some things which weren't working for me :

- I will just add some weird things which came after flashing the rom
![Screenshot_20211213-000342_Setup_Wizard](https://user-images.githubusercontent.com/97163340/148204291-ec34e3b5-55d6-49de-88c6-2914ec11792c.png)
![Screenshot_20220105-155203_DivestOS_Settings](https://user-images.githubusercontent.com/97163340/148204342-5c819c9f-3c53-490d-8bef-43535b164a2a.png)
![Screenshot_20220105-155402_F-Droid](https://user-images.githubusercontent.com/97163340/148204394-d2d285e2-0896-43ed-b625-3595ea682471.png)

while I understand they aren't really an issue , can something be done for it so that these message pop up don't come .
Also
 
 I haven't tried relocking using custom avb but will soon :)
 
 I also wanted to ask why silence is still the default message app . Is there some specific reason for it ?
 
 If there's any way I can help , please do tell me

**Comment by SkewedZeppelin** [comment, marked as answer]
So those warnings are expected, and show for any app not targeting Android 9.0 or higher.
It is to warn the user, as such apps have a slightly less restrictive sandbox applied to them from what I understand.

I can't remove Silence, because anyone who uses it will have all their messages deleted.
I will likely drop Silence from my 12.0/19.0 builds and make a very bold banner informing users to backup messages.

Otherwise, I'm glad to see 7 series booting! It was previously marked broken.
Thank you for testing!

**Comment by ghost** [comment]
Btw . I have found a weird thing 

If I flash  the rom after the stock rom  , the sim doesn't get detected  however if I flash  the rom after flashing a custom rom  ( for ex yaap ) the sim does get detected without any issues . 

I don't know why this is happening 


Also , the bootloader locking with custom avb key works 😃

**Reply by SkewedZeppelin** [reply]
The modem/radio issue has been tracked down, should be fixed in next build.

**Comment by SkewedZeppelin** [comment]
Are you on the absolute latest firmware?
Perhaps the other ROM has firmware included.
DivestOS currently doesn't include firmware for these devices.

**Reply by ghost** [reply]
ah yes.

I guess it then makes sense why the latest lineage os or others are not having such issue .

**Reply by ghost** [reply]
ok so here's an interesting thing I found . 

Both the lineage os and in what I used before ( yaap ) don't actually have the firmware files in their zips 😕 
So now I don't know what is happening

**Comment by SkewedZeppelin** [comment]
Please try clearing app data of Silence.
There is no need to reset your whole device.

**Reply by Guardian-Dusty** [reply]
👍
